app/assets/images/payment_icons/qris.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" role="img" viewBox="0 0 38 24" width="38" height="24" aria-labelledby="pi-qris"><title id="pi-qris">QRIS</title><path fill="#000" opacity=".07" d="M35 0H3C1.3 0 0 1.3 0 3v18c0 1.7 1.4 3 3 3h32c1.7 0 3-1.3 3-3V3c0-1.7-1.4-3-3-3z"/><path fill="#fff" d="M35 1c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32"/><g clip-path="url(#clip0)" fill="#010101"><path d="M34.607 13.678V10.445H29.21V9.367h5.397V7.212h-8.635V12.6h5.398v1.078h-5.398v2.155h8.635v-2.155zM24.893 7.212h-2.159v8.62h2.16v-8.62zM13.02 7.212v2.155h6.476v1.078H13.02v5.388h2.158v-3.2l3.24 3.2h3.237L18.277 12.6h3.378V7.212h-8.634zM6.543 12.6h2.154v-2.15H6.543v2.15zm.541-1.616h1.079v1.077H7.084v-1.077z"/><path d="M5.465 7.212h-1.89a.273.273 0 00-.249.167.269.269 0 00-.02.103v8.08a.269.269 0 00.27.27h5.127v-2.15H5.465v-6.47zM11.67 7.212H6.544v2.155h3.24V12.6h2.153V7.482a.273.273 0 00-.076-.191.277.277 0 00-.19-.08zM11.942 13.678h-2.16v4.85h2.16v-4.85z"/><path d="M6.318 6H2.27a.271.271 0 00-.27.27v4.04h.54V6.804a.27.27 0 01.27-.264h3.513L6.318 6zM35.46 13.004v3.507a.27.27 0 01-.27.27h-3.512v.534h4.048a.273.273 0 00.253-.166.27.27 0 00.021-.104v-4.041h-.54z"/></g><defs><clipPath id="pi-qris-clip0"><path fill="#fff" transform="translate(2 6)" d="M0 0h34v12.526H0z"/></clipPath></defs></svg>
```

Comment:
Correct me if I'm wrong but it doesn't seems like we need the `clipPath`
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" role="img" viewBox="0 0 38 24" width="38" height="24" aria-labelledby="pi-qris"><title id="pi-qris">QRIS</title><path fill="#000" opacity=".07" d="M35 0H3C1.3 0 0 1.3 0 3v18c0 1.7 1.4 3 3 3h32c1.7 0 3-1.3 3-3V3c0-1.7-1.4-3-3-3z"/><path fill="#fff" d="M35 1c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32"/><g clip-path="url(#clip0)" fill="#010101"><path d="M34.607 13.678V10.445H29.21V9.367h5.397V7.212h-8.635V12.6h5.398v1.078h-5.398v2.155h8.635v-2.155zM24.893 7.212h-2.159v8.62h2.16v-8.62zM13.02 7.212v2.155h6.476v1.078H13.02v5.388h2.158v-3.2l3.24 3.2h3.237L18.277 12.6h3.378V7.212h-8.634zM6.543 12.6h2.154v-2.15H6.543v2.15zm.541-1.616h1.079v1.077H7.084v-1.077z"/><path d="M5.465 7.212h-1.89a.273.273 0 00-.249.167.269.269 0 00-.02.103v8.08a.269.269 0 00.27.27h5.127v-2.15H5.465v-6.47zM11.67 7.212H6.544v2.155h3.24V12.6h2.153V7.482a.273.273 0 00-.076-.191.277.277 0 00-.19-.08zM11.942 13.678h-2.16v4.85h2.16v-4.85z"/><path d="M6.318 6H2.27a.271.271 0 00-.27.27v4.04h.54V6.804a.27.27 0 01.27-.264h3.513L6.318 6zM35.46 13.004v3.507a.27.27 0 01-.27.27h-3.512v.534h4.048a.273.273 0 00.253-.166.27.27 0 00.021-.104v-4.041h-.54z"/></g></svg>
```

Reply:
you are correct.
thanks.

Reply:
@tauthomas01 updated

Reply:
It works! Thank you for making the changes.